clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -61,8 +61,9 @@ func (p *profileMeasurement) populateProfileConfig(config *measurement.Measureme
 	if p.config.provider, err = util.GetStringOrDefault(config.Params, "provider", config.ClusterFramework.GetClusterConfig().Provider); err != nil {
 		return err
 	}
-	if p.config.host, err = util.GetStringOrDefault(config.Params, "host", config.ClusterFramework.GetClusterConfig().GetMasterIp()); err != nil {
-		return err
+	p.config.hosts = config.ClusterFramework.GetClusterConfig().MasterIPs
+	if len(p.config.hosts) < 1 {
```

Comment:
Can we move this to the start() method and exit in that case? 
There is no point in executing any other code if there are no hosts to scrape the profiles from :)

Reply:
Done. thank you for pointing this out :sweat_smile: